src/preamble.js

```diff
@@ -802,14 +801,81 @@ var wasmOffsetConverter;
 #if SPLIT_MODULE
 var splitModuleProxyHandler = {
   'get': function(target, prop, receiver) {
-    err('placeholder function created: ' + prop);
     return function() {
-      abort('placeholder function called: ' + prop);
+      err('placeholder function called: ' + prop);
+      var imports = {'primary': Module['asm']};
+      var table = Module['asm']['__indirect_function_table'];
+      instantiateSync(wasmBinaryFile + '.deferred', imports);
```

Comment:
I love that you don't even need to receive handles to the module be instantiated here!  Its completely self contained.